### ISSUE TITLE
Sleep modes

### ISSUE THREAD

**Issue by yomasa** [espressif/arduino-esp32#380]
Can someone point me to any code examples of Sleep modes with rtc?

**Comment by copercini** [comment]
There are some examples of Deep Sleep with RTC support: https://github.com/espressif/arduino-esp32/tree/master/libraries/ESP32/examples/DeepSleep

**Comment by beats0126** [comment]
@copercini may i ask for the deep sleep problem, i using the example TouchWakeUp to do the deep sleep. i set the pin 15 as pin to call esp32 wakeup. but the problem i meet is it keep wakeup after going sleep. i am a beginner to this field. for checking the problem, i removed all wire from the board and only connect 1 f2f to my esp32 (espduino32), I also found if my finger touch pin 15 it also causing esp32 wake up. is it the logic to wakeup the esp32 when it receive a small power from the pin we selected?
